### PR TITLE
[Fix] Update faker package reference

### PR DESCRIPTION
**Pull request by studioswong**
## Description

Resolves  issue #738.

This is to fix the errors with resolving the `faker` dependency when Kedro-Viz is imported as an npm package. 

## Development notes

This error is caused by the packages being declared under `dev-dependencies` instead of `dependencies`, with which I had updated the packages in our `package.json` file. 

I had also updated the reference of the `faker` npm package to `@faker-js/faker` ([see here](https://www.npmjs.com/package/@faker-js/faker)). 

## QA notes

N/A

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/784"></a>

### CODE REVIEW

**Comment by tynandebold** [comment]
What do you think about taking this one step further and completely removing these two dependencies and everything that comes into contact with them?

The reason we have them is for the GraphQL mocks, yet we don't use the mocks for any testing or anything else. And if we did, they don't accurately represent what experiment tracking looks like since there isn't any tracking data or metadata, as shown here when the mocks are toggled to `true`:

Screen Shot 2022-03-11 at 15 40 57

Lastly, now that we have the `demo-project` integrated into the repo, that's effectively mocking experiment tracking, and working quite well at that, so this seems like a great opportunity to trim down some unused aspects of the codebase.

**Comment by studioswong** [comment]
yea I thought about that too, and I agree the mocks haven't been updated when we have the backend endpoints.

That said, I do still think there is value in keeping the mocks given its use as a development tool to help us quickly whip up some data without relying on the Backend or for quickly validating some data set up before investing in the backend - I can see this set up still being of use in the future. 

The situation with the demo project that you described here is a different scenario where it requires being spun up as a dev server, and can only serve the data that is returned by the existing backend setup, which is different from the set up of this mock, being completely independent as a FE tool with its intention to mock any potential new BE endpoint before its development. 

As long as this fix solves the error then we should keep this.

File: README.md [hunk 1]
```diff
@@ -59,7 +59,17 @@ There are two ways you can use Kedro-Viz:
...
+
+    ```javascript
+    <div style={{ height: "100vh" }}>
+      <KedroViz
+        data={dataPipeline}
+      />
+    </div>
+    ```
+  
+  Please ensure that the relevant pipeline data is passed as a prop into the component. We also recommend wrapping the `Kedro-Viz` component with a parent HTML/JSX element that has a specified height (as seen in the above example) in order for Kedro-Viz to be styled properly.
```
Comment (tynandebold) [inline README.md:72]: We already have a section in the readme for the [standalone component usage](https://github.com/kedro-org/kedro-viz#standalone-react-component-usage). 

You can probably update the second line in the example already in that section to this:

```jsx
const MyApp = () => (
  <div style={{ height: "100vh" }}>
    <KedroViz data={json} />
  </div>
);
```

And then add the note:

> We recommend wrapping the `Kedro-Viz` component with a parent element that has a specified height in order for Kedro-Viz to be styled properly.

**Review by tynandebold — APPROVED** [review]
Looks good!

Just one thing about updating the readme: we already have a section for standalone component usage so it's probably better to just update the content there.

I'll approve straight away though, as that's my only comment.

File: README.md (hunk 1 above)
Comment (studioswong) [reply README.md]: good spot - completely overlooked the section below.

I've updated the section and wording accordingly.

**Pull request merged**